app/src/main/resources/config/leagur-service.conf

```diff
@@ -5,3 +5,9 @@
 # classes defined within the Governator dependency
 # injection framework.
 #
+
+### Store configuration ###
+hyperdex.store.endpoint=${HYPERDEX_ENDPOINT}
+# Use 1982 as the default port
+hyperdex.store.port=1982
+hyperdex.store.port=${?HYPERDEX_PORT}
```

Comment:
This is the syntax for setting a default value. the `?` means get the environment variable if it exists. If it does not, then the configuration will be set to 1982.